### ISSUE TITLE
Listen for deletions

### ISSUE THREAD

**Issue by jo** [eHealthAfrica/kazana-transform-listener#4]

**Comment by jo** [comment]
@tlvince could you specify what you need here?

**Comment by tlvince** [comment]
We need to know when a deletion has finished on the [email box store](https://github.com/eHealthAfrica/van/blob/0427eac6a16f162836b48c6c322913c569af5d8a/lib/routes/inbox-api.js#L68-L85).

**Comment by jo** [comment]
ah. Not in raw data?

**Comment by tlvince** [comment]
Nope. (Nothing gets deleted from raw data AFAIK?). Would it make sense to make this repos store configurable?

**Comment by jo** [comment]
At least we have a route for the deletion: `DELETE /api/submissions/{id}`

**Comment by jo** [comment]
I think it makes sense to listen to raw-data-deletions anyway: https://github.com/eHealthAfrica/kazana-transform-listener/pull/5

**Comment by jo** [comment]
I mean #5 does not solve your actual issue
